tests/conftest.py

```diff
@@ -106,9 +106,14 @@ def fast(request):
 
 def pytest_generate_tests(metafunc):
     # setup config fixture to get all of the results from config_generator
+    def make_test_id(config):
+        return f"{config['environment_manager']}-{config['dependency_file']}-{config['docs']}"
```

Comment:
I think this would be more informative if we named the config with whether it contained the basic pydata packages rather than docs, since the former has more impact than the latter.
```suggestion
        return f"{config['environment_manager']}-{config['dependency_file']}-{config['pydata_packages']}"
```